scalableminds/webknossos
# Make tables horizontally scrollable on overflow and add wordbreak in tables

This PR adds CSS rule to make tables horizontally scrollable when the screen/container of the table is not wide enough to display the whole table. It also adds the word-break property to the table data to force long words in a table cell to be written in multiple lines.

Additionally, this PR adds the possibility of clicking on the dataset name in the dataset table (dashboard) to open the dataset in view mode.

### URL of deployed dev instance (used for testing):
- https://maketablesscrollable.webknossos.xyz
### Steps to test:
- Open the dataset table in the dashboard. 
- Resize the window to have a small width. The dataset table should now be horizontally scrollable. The scrollbar should be at the bottom of the view. Long dataset names should also be displayed in multiple lines. 
- Click on the name of a dataset. It should open up the dataset in view mode.

### Issues:
- fixes #4106 

------
